lang/pt_BR.json

```diff
@@ -1,11 +1,407 @@
+    "Site Msg": "Mensagem no Site",
+    "Site Moderator": "Moderador do Site",
+    "Site Preferences": "Preferências do Site",
+    "site-preferences": "Preferências do Site",
+    "Site Rank:": "Rank do Site",
+    "Someone follows me:": "Alguém me Seguir",
+    "started playing a game": "começou a jogar um jogo",
+    "Subscribe to comments": "Inscreva-se para os comentários",
+    "System Default": "Padrão do Sistema",
+    "The Connect Key is used in emulators to keep you logged in. Resetting the key will log you out of all emulators.": "A Chave de Conexão é utilizada nos emuladores para manter você conectado. Redefinir a Chave vai fazer com que você seja desconectado de todos os emuladores.",
+    "The Golden Era of Handheld Gaming - whether it's Tetris, MArio or Pokemon, discuss your favourite GB/GBA titles here!": "A Época de Ouro dos Consoles Portáteis - seja Tetris, Mario ou Pokemon, discuta seus títulos favoritos de GB/GBA aqui!",
+    "There are :numberOfFilteredPlayers active players.": "Existem :numberOfFilteredPlayers jogadores ativos."
+    "There are :numberOfFilteredPlayers filtered active players (out of :numberOfPlayersActive total).": "Existem :numberOfFilteredPlayers jogadores ativos filtrados (De um total de :numberOfPlayersActive).",
+    "terms-developers": "Código de Conduta dos Desenvolvedores",
+    "This is your personal Web API Key. Handle it with care.": "Essa é sua Chave Web API pessoal. Guarde-a com segurança.",
+    "Tickets Resolved For Others": "Tickets Resolvidos para Outros",
+    "to find someone to add to follow?": "para achar alguém para seguir?",
+    "Total Points": "Total de Pontos",
+    "Unsubscribe from comments": "Desinscreva-se dos Comentários",
+    "User Documentation": "Documentação de Usuários",
+    "User List": "Lista de Usúarios",
+    "User List:": "Lista de Usuários:",
+    "User Motto": "Lema do Usuário",
+    "User Wall": "Parede de Usuários",
+    "Users Code of Conduct": "Código de Conduta dos Usuários",
+    "unlocked an achievement": "desbloqueou uma conquista",
+    "username-change": "Mudar nome de Usuário",
+    "View All User Set Requests": "Ver Todas as Coleções Pedidas por Usuários",
+    "View Unread Only": "Ver Somente Não-Lidas",
+    "wall comment": "Parede de Comentário",
+    "wall comments": "Parede de Comentários",
+    "What was your favourite SNES game? Post here about SNES games and about the RASnes9x app.": "Qual era seu jogo de SNES favorito? Publique aqui sobre jogos de SNES e sobre o aplicativo do RASnes9x.",
+    "Web API Key": "Chave Web API",
+    "Weekly RA Newsletter": "Notícias RA Semanais",
+    "Yielded Unlocks": "Desbloqueios Providos",
+    "Yielded Points": "Pontos Providos",
+    "You don't appear to be following anyone yet. Why not": "Você não parece estar seguindo ninguém ainda. Porquê você não",
+    "Your motto": "Seu Lema",
+    
+    ///////Translation made by:///////
+    //////////////Elfocat/////////////
```

Comment:
While your intention to get credit for your work is understandable, such information does not belong in an i18n file.

This is to maintain the clarity and maintainability of the codebase, and to ensure consistency with the GNU General Public License 3.0 under which the project operates. This license allows anyone to use, modify, and distribute the code, so individual contributor attribution in the code itself is not necessary or appropriate. Our version control system automatically keeps track of contributions,